Clockify.Net/Models/Reports/ProjectFilterDto.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace Clockify.Net.Models.Reports
+{
+    public class ProjectFilterDto
```

Comment:
The payload of this dto is compatible with the payload frontend is sending:

![image](https://user-images.githubusercontent.com/9637281/200105877-0cf57b4d-e3d2-41d9-a223-0f557c656b5d.png)